src/components/Accordion/__test__/accordion.spec.js

```diff
@@ -0,0 +1,399 @@
+
+        expect(handleToggleSectionMockFn.mock.calls[0][1]).toEqual('');
+    });
+    it('should call handleToggleSection with correct "argument" when click in the "second AccordionSection", "multiple" is passed and both "AccordionSection" are expanded', () => {
+        const handleToggleSectionMockFn = jest.fn();
+        const activeNames = ['accordion-test-1', 'accordion-test-2'];
+        const component = mount(
+            <div>
+                <Accordion
+                    multiple
+                    activeSectionNames={activeNames}
+                    onToggleSection={handleToggleSectionMockFn}>
+                    <AccordionSection
+                        name="accordion-test-1"
+                        label="Rainbow Accordion">
+                        AccordionSection-1
+                    </AccordionSection>
+                    <AccordionSection
+                        name="accordion-test-2"
+                        label="Rainbow Accordion">
+                        AccordionSection-2
+                    </AccordionSection>
+                </Accordion>
+            </div>,
+        );
+        const secondAccordionSection = component.find('ul').childAt(1);
+        const secondAccordionSectionButton = secondAccordionSection.find('button.rainbow-accordion-section_summary-button-heading');
+        secondAccordionSectionButton.simulate('click');
+
+        expect(handleToggleSectionMockFn.mock.calls[0][1]).toEqual(['accordion-test-1']);
+    });
+    it('should call handleToggleSection with correct "argument" when click in the "second AccordionSection", it is collapse for the first time and "multiple" is passed', () => {
+        const handleToggleSectionMockFn = jest.fn();
+        const component = mount(
+            <div>
+                <Accordion
+                    multiple
+                    onToggleSection={handleToggleSectionMockFn}>
+                    <AccordionSection
+                        name="accordion-test-1"
+                        label="Rainbow Accordion">
+                        AccordionSection-1
+                    </AccordionSection>
+                    <AccordionSection
+                        name="accordion-test-2"
+                        label="Rainbow Accordion">
+                        AccordionSection-2
+                    </AccordionSection>
+                </Accordion>
+            </div>,
+        );
+        const secondAccordionSection = component.find('ul').childAt(1);
+        const secondAccordionSectionButton = secondAccordionSection.find('button.rainbow-accordion-section_summary-button-heading');
+        secondAccordionSectionButton.simulate('click');
+
+        expect(handleToggleSectionMockFn.mock.calls[0][1]).toEqual(['accordion-test-2']);
+    });
+
+/* when any <AccordionSection /> is expanded by default */
+    it('should set aria-expanded to "true" in "button" when click in AccordionSection', () => {
```

Comment:
change this test to not use click only pass the right activeSectionNames and check that is expanded, also change this name because there is not more click.